inputfiles/addedTypes.jsonc

```diff
@@ -449,6 +449,40 @@
     },
     "interfaces": {
         "interface": {
+            // ImportMeta is not a true DOM interface, but we are forced to declare it as one in order to emit method definitions.
+            // We cannot define methods as dictionary properties with function types,
+            // as this would cause conflicts with ImportMeta method overrides in places like @types/node.
+            "ImportMeta": {
+                "name": "ImportMeta",
+                "exposed": "Window Worker",
+                "noInterfaceObject": true,
+                "properties": {
+                    "property": {
+                        "url": {
+                            "name": "url",
+                            "type": "DOMString"
+                        }
+                    }
+                },
+                "methods": {
+                    "method": {
+                        "resolve": {
+                            "name": "resolve",
+                            "signature": [
+                                {
+                                    "param": [
+                                        {
+                                            "name": "moduleName",
```

Comment:
One nit, as the spec says "specifier", can you also match this param name with that?

> 4. Let steps be the following steps, given the argument *specifier*:

Reply:
I went with MDN's nomenclature, but happy to go off the spec if that's preferable.

Reply:
I use it to find the folder of the script via
`import.meta.resolve("./")`
So IMHO both are not always matching but `specifier` is generic enough.